src/coprocessor/dag/rpn_expr/types/expr_eval.rs

```diff
@@ -258,13 +267,17 @@ impl RpnExpression {
                     let stack_slice_begin = stack.len() - func.args_len();
                     let stack_slice = &stack[stack_slice_begin..];
                     let call_info = RpnFnCallPayload {
+                        output_rows,
                         raw_args: stack_slice,
                         ret_field_type: field_type,
                     };
-                    let ret = func.eval(rows, context, call_info)?;
+                    let ret = func.eval(context, call_info)?;
```

Comment:
The payload becomes too big to copy. We can use its reference instead :)
(I can do it in my RPN function PR)

Reply:
Agree.